CHANGELOG.asciidoc

```diff
@@ -36,6 +36,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...master[Check the HEAD d
 *Auditbeat*
 
 *Filebeat*
+- Correctly parse `December` or `Dec` in the Syslog input. {pull}xxx[xxx]
```

Comment:
@ph I am adding this missing reference to PR in #9358 along with another one I added.

Reply:
Thanks I have fixed all the other backport to include the reference.